### PR TITLE
Iteration on support page

### PR DESCRIPTION
**Pull request by Yndira-FlowForge**
## Description

Updated content and layout

## Related Issue(s)

#720

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/support.njk [hunk 1]
```diff
@@ -1,27 +1,114 @@
 ---
-layout: layouts/nohero.njk
-title: Request Support
+align: center
+layout: layouts/page.njk
+title: We’re here to help!
```
Comment (ZJvandeWeg) [inline src/support.njk:4]: Is the title element in the HEAD of the HTML the same? In that case I'd rather have Support

File: src/support.njk [hunk 2]
```diff
@@ -1,27 +1,114 @@
...
+              </div>
+            </h3>
+            <p class="mb-0">
+              Get real-time assistance and quick answers. Chat with us for prompt support and solutions to your queries.
+            </p>
+          </div>
+          <div class="container text-center max-w-4xl flex flex-col md:items-start mt-6">
+            <button onclick="window.HubSpotConversations.widget.open()" class="ff-btn ff-btn--primary flex flex-col">
+              <span class="text-base uppercase no-underline md:px-4">
+                open chat box
```
Comment (ZJvandeWeg) [inline src/support.njk:64]: Nit: COMMUNITY FORUM is in all caps too

```suggestion
                OPEN CHAT BOX
```

**Review by ZJvandeWeg — APPROVED** [review]

File: src/support.njk (hunk 1 above)
Comment (Yndira-FlowForge) [reply src/support.njk:4]: Yeah, it's the same... Will change it back to Support

File: src/support.njk (hunk 2 above)
Comment (Yndira-FlowForge) [reply src/support.njk:64]: The "uppercase" class takes care of that ;) 
Will change it anyways just to tidy it up

File: src/support.njk (hunk 1 above)
Comment (ZJvandeWeg) [reply src/support.njk:4]: Help center is fine with me

**Pull request merged**